docker-compose.yaml

```diff
@@ -7,7 +7,8 @@ services:
       context: "."
     image: "extra-model-dev"
     volumes:
-      - ./:/io
+      - ./extra_model:/io/extra_model
```

Comment:
The only other thing that I'm sure we need to have is embeddings. Not sure what's the good way to mount them though. In principle, the folder with embeddings can be named anything, so not sure how we can generalize.
In the meantime probably still good idea to have `./glove_embeddings:/io/glove_embeddings` here.

Reply:
at least once we have script to seed container with data we can enforce the naming of the folder better

Reply:
actually, I thought of another case that we need to have - mounting input files and outputting files.

Reply:
cool, thanks for these points. I'd prefer to address mounting of the seed data on the next PR which implements the "setup script" for downloading and formatting the glove embeddings. I'm making a change now for the user input / output when running using `docker-compose`.